rust/automerge/src/types.rs

```diff
@@ -158,6 +159,12 @@ pub enum ObjType {
     Text,
 }
 
+impl Default for ObjType {
```

Comment:
This seems surprising to me. Do we need a `Default` implementation for `ObjType`? I'm very suspicious of `Default` implementations which are not isomorphic to some kind of "zero" value, which `ObjType` doesn't have (to my mind anyway).

Reply:
Nope.  Whatever I needed that for was removed.